cmd/ttn-lw-cli/commands/packetbroker.go

```diff
@@ -149,6 +149,43 @@ func getPacketBrokerRoutingPolicy(flagSet *pflag.FlagSet) (up ttnpb.PacketBroker
 	return
 }
 
+func packetBrokerGatewayVisibilityFlags() *pflag.FlagSet {
+	flagSet := &pflag.FlagSet{}
+	flagSet.Bool("location", false, "")
+	flagSet.Bool("antenna-placement", false, "")
+	flagSet.Bool("antenna-count", false, "")
+	flagSet.Bool("contact-info", false, "")
+	flagSet.Bool("status", false, "")
+	flagSet.Bool("frequency-plan", false, "")
+	flagSet.Bool("packet-rates", false, "")
+	flagSet.Bool("all", false, "enable all fields")
+	return flagSet
+}
+
+func getPacketBrokerGatewayVisibility(flagSet *pflag.FlagSet) *ttnpb.PacketBrokerGatewayVisibility {
+	res := &ttnpb.PacketBrokerGatewayVisibility{}
+	res.Location, _ = flagSet.GetBool("location")
+	res.AntennaPlacement, _ = flagSet.GetBool("antenna-placement")
+	res.AntennaCount, _ = flagSet.GetBool("antenna-count")
+	res.ContactInfo, _ = flagSet.GetBool("contact-info")
+	res.Status, _ = flagSet.GetBool("status")
+	res.FrequencyPlan, _ = flagSet.GetBool("frequency-plan")
+	res.PacketRates, _ = flagSet.GetBool("packet-rates")
+
+	if all, _ := flagSet.GetBool("all"); all {
+		res.Location = true
+		res.AntennaPlacement = true
+		res.AntennaCount = true
+		res.ContactInfo = true
+		res.Status = true
+		res.FrequencyPlan = true
+		res.PacketRates = true
+	}
+	return res
+}
```

Comment:
We have utilities that can extract this from and set this to the proto message, why not use those?

Reply:
This is probably for consistency reasons, given that the whole file uses this style.